orttraining/orttraining/python/training/ortmodule.py

```diff
@@ -436,8 +436,8 @@ def _get_inference_graph(self, *inputs, **kwargs):
         dynamic_axes.update(output_dynamic_axes)
 
         # TODO: Support contrib OPs support? user model has no hint
-        # from onnxruntime.training import register_custom_ops_pytorch_exporter
-        # register_custom_ops_pytorch_exporter.register_custom_op()
+        from onnxruntime.training import register_custom_ops_pytorch_exporter
```

Comment:
why not put this at the very top. Considering that this will be called at least once on model export with ```ORTModule```, we can always import it?